src/Ryujinx.Horizon/Sdk/Settings/System/DeviceNickName.cs

```diff
@@ -0,0 +1,9 @@
+using System.Runtime.InteropServices;
+
+namespace Ryujinx.Horizon.Sdk.Settings.System
+{
+    [StructLayout(LayoutKind.Sequential, Size = 0x80)]
+    struct DeviceNickName
+    {
+    }
```

Comment:
```suggestion
    [StructLayout(LayoutKind.Sequential, Size = 0x80)]
    public struct DeviceNickName
    {
        public ByteArray128 Value;

        public DeviceNickName(string value)
        {
            byte[] buffer = Encoding.ASCII.GetBytes(value);

            Array.Resize(ref buffer, 0x80);

            if (buffer.Length > 0x80)
            {
                buffer[0x80] = 0;
            }
            else
            {
                buffer[value.Length] = 0;
            }

            buffer.AsSpan().CopyTo(Value.AsSpan());
        }
    }
```

Part of some old WIP code